jodd-mail/src/main/java/jodd/mail/EmailAttachment.java

```diff
@@ -284,6 +284,7 @@ public void writeToFile(final File destination) {
 		}
 		finally {
 			StreamUtil.close(input);
+			StreamUtil.close(output);
```

Comment:
Hi,

your change will produce a compiler error - see [travis](https://travis-ci.org/oblac/jodd/jobs/632646285).

this should work:

```
	public void writeToFile(final File destination) {
		InputStream input = null;
		OutputStream output = null;
		try {
			input = getDataSource().getInputStream();
			output = new FileOutputStream(destination);

			StreamUtil.copy(input, output);
		}
		catch (final IOException ioex) {
			throw new MailException(ioex);
		}
		finally {
			StreamUtil.close(input);
			StreamUtil.close(output);
		}
	}
```

Reply:
Thanks!